app/redux/ui/actions.ts

```diff
@@ -14,7 +14,11 @@ export const setOsTheme = () => async (dispatch: AppThDispatch) => {
   dispatch({ type: SET_OS_THEME, payload: { isDarkTheme } });
```

Comment:
Now it looks complicated and unclear:
what is the difference between `setOsTheme` and `switchTheme`.
If we take a look on implementation and consumers it seems like:
- `setOsTheme` — requests for `isDarkMode` from main process without switching
- `switchTheme` — switches is and sends a new value to the main process, which will be updated in persistent store as well

So I propose to rename: `setOsTheme` -> `loadTheme` and leave `switchThem` as is.

Reply:
Makes sense.